hw/sensor/src/sensor.c

```diff
@@ -1970,14 +1970,19 @@ sensor_set_thresh(const char *devname, struct sensor_type_traits *stt)
     struct sensor *sensor;
     int rc;
 
+    if (stt != NULL) {
```

Comment:
so now I am confused... we want to avoid null dereference or force it? :D

also update commit message

Reply:
avoid null, of course... :)
updated